rootfs/ingest/python/qserv/replicationclient.py

```diff
@@ -46,7 +46,7 @@
 # Local non-exported definitions --
 # ---------------------------------
 _LOG = logging.getLogger(__name__)
-_VERSION = 9
+_VERSION = 11
```

Comment:
Is there version 10 already on some other branch?

Reply:
I'll rename _VERSION to _REPL_SERVICE_VERSION for clarity